zenodo/modules/deposit/static/json/zenodo_deposit/deposit_form.json

```
@@ -527,6 +527,14 @@
                 },
                 "default_funder": "10.13039/501100000780",
                 "funders_map": [
+                  {
+                    "value": "10.13039/501100002341",
+                    "name": "Academy of Finland (FI)"
+                  },
+                  {
+                  "value": "10.13039/501100001665",
+                  "name": "Agence Nationale de la Recherche (FR)"
```

```suggestion
                    "value": "10.13039/501100001665",
                    "name": "Agence Nationale de la Recherche (FR)"
```

For readability, let's keep the alignment the same across all the options